index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="topnav">
+            <nav class="navi">
+                <ul>
+                    <!-- <li><a href="#" class="logo">Mucha Naibei</a></li> -->
+                    <li><a href="#" class="logo"><img src="images/avatar.png" alt=""></a></li>
+                    <!-- <li><a class="hamburger" href="#navbar">
+                            <img src="https://ljc-dev.github.io/testing0/ham.svg" alt="menu"></a></li> -->
+                    <li><a href="#"><img src="images/Union.png" alt="Union"></a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <div class="intro">
+            <h1>I'm Mucha <br> Glad to see <br> you!</h1>
+            <p>I'm a software developer! I can help you <br> build a product, feature or website Look <br> through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't <br> hestiate to contact me.
+            </p>
+
+            <a href="#" class="con">LET'S CONNECT</a>
+
+            <ul class="socials">
+                <li><a href="#"><img src="images/github.svg" alt="Github"></li>
+                <li><a href="#"><img src="images/Linkedin icon.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+                <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"> </a></li>
+            </ul>
+        </div>
+    </div>
```

Comment:
- You have done a good job so far, but there are some differences between your project and the Figma design template. Please fix the changes 🙏 

Your project:
-- There is too much space between the navbar and the intro section as marked in your design. Please fix this 🙏 
-- Also please ensure the background image for this section is properly aligned as it is in the design template 👍 
<img width="230" alt="ggwrgwrg" src="https://user-images.githubusercontent.com/73607512/219125607-f26b5ffe-25e1-4421-a9ad-248f5511ef95.PNG">
Figma template:
<img width="142" alt="header portfolio 2" src="https://user-images.githubusercontent.com/73607512/219126583-8ede86fb-55a8-4292-8df3-45bc14eb9c7b.PNG">